client/src/components/profileInfoEdit/ProfileResetModal.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Delete Alert Modal component display a modal and ask for confirmation when the delete button is clicked
+ */
+
+import React from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { logout } from '../../actions/auth';
+import { resetTeamData, deleteGHPages } from '../../actions/team';
+import { resetHomepage } from '../../actions/homepage';
+import { resetWebPage } from '../../actions/website';
+
+const ProfileResetModal = ({ resetAlert, setResetAlert, type }) => {
+  const { teamId } = useSelector((state) => state.team);
+  const dispatch = useDispatch();
+
+  const isDeleteFlag = type;
+  const titleMessage = isDeleteFlag ? 'Delete Account Data!' : 'Clear Account Data!';
+  const warningMessage = isDeleteFlag ? 'Are you sure you want to delete your account? ' : 'Are you sure you want to reset your account? ';
+  const buttonName = isDeleteFlag ? 'Delete' : 'Clear';
+
+  const HandleResetType = () => {
+    const accessToken = localStorage.getItem('GH_access_token');
```

Comment:
What happens if the access token is not found in localStorage? See my comment earlier: https://github.com/Researchify/Researchify/pull/198/files#r711673343

Reply:
IF the token is not found it is being handled in the front end and the GhLoginModal gets triggered which prompts the user to log in with their github